the-journals-of-gerontology-series-a.csl

```diff
@@ -0,0 +1,260 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US" page-range-format="expanded">
+  <info>
+    <title>The Journals of Gerontology: Series A</title>
```

Comment:
It's a small nitpick, but I'd rather see "The Journals of Gerontology, Series A" (comma instead of colon) or the full "The Journals of Gerontology, Series A: Biological Sciences and Medical Sciences" as the title, as used at http://www.oxfordjournals.org/our_journals/gerona/for_authors/general.html.